### PR TITLE
MOCK: Update get started to latest

### PR DESCRIPTION
**Pull request by philschanely**
## Description

This PR brings the Get started mocks in React up to date with latest work from related PRs,

## Testing in `sage-lib`

See http://localhost:4100/?path=/story/mocks-get-started--dashboard-module

## Testing in `kajabi-products`
1. (N/A) Update to Get Started Mocks

### CODE REVIEW

File: packages/sage-react/lib/themes/legacy/mocks/get-started/components/GetStartedTabs.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  return (
+    <Frame direction={Frame.DIRECTIONS.HORIZONTAL} gap={Frame.GAPS.LG}>
+      <Frame width="240px" gap={Frame.GAPS.SM}>
+        {tabsContent.map(({
+          completed = false,
+          icon,
+          id,
+          label,
+        }) => (
+          <Tabs.Item
```
Comment (monkeypox8) [inline packages/sage-react/lib/themes/legacy/mocks/get-started/components/GetStartedTabs.jsx:24]: We may need to revisit the assignment logic of `TagName` in the `Tabsitem` component as a follow-up. These tabs do not receive keyboard focus as `div`s, so we should consider defaulting to `button` instead, which matches the Rails component.

File: packages/sage-react/lib/themes/legacy/mocks/get-started/components/GetStartedTabsMobile.jsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      id={id}
+      padding={Frame.PADDINGS.SM}
+      width={Frame.WIDTHS.FILL}
+    >
+      <Frame
+        align={Frame.ALIGNMENTS.CENTER_SPREAD}
+        direction={Frame.DIRECTIONS.HORIZONTAL}
+        id={`${id}-control`}
+        onClick={() => setOpen(!open)}
+        role="button"
```
Comment (monkeypox8) [inline packages/sage-react/lib/themes/legacy/mocks/get-started/components/GetStartedTabsMobile.jsx:32]: To take this this thought further, you might consider replacing this role with `tag="button"`, and setting the ensuing child `Frame` components to `tag="span"`

File: packages/sage-react/lib/themes/legacy/mocks/get-started/GetStarted.story.jsx [hunk 3]
```diff
@@ -9,12 +13,17 @@ export default {
   argTypes: {},
   args: {}
 };
-const Template = () => <DashboardPanel />;
+
+const Template = () => (
+  <SageBreakpointProvider queries={SageTokens.BREAKPOINT_QUERIES}>
```
Comment (monkeypox8) [inline packages/sage-react/lib/themes/legacy/mocks/get-started/GetStarted.story.jsx:18]: Oooh, verry interesting 😍

**Review by monkeypox8 — APPROVED** [review]
👍 LGTM! Added some comments to consider around enhancements to choice accessibility.

File: packages/sage-react/lib/themes/legacy/mocks/get-started/components/GetStartedTabs.jsx (hunk 1 above)
Comment (philschanely) [reply packages/sage-react/lib/themes/legacy/mocks/get-started/components/GetStartedTabs.jsx:24]: FWIW Added Jira issue https://kajabi.atlassian.net/browse/SAGE-623

**Pull request merged**